<launch>
    <arg name="pkg" value="quickdev_examples" />
    <arg name="type" value="multi_publisher" />
    <arg name="name" default="$(arg type)" />
    <arg name="rate" default="30" />
    <arg name="args" value="_loop_rate:=$(arg rate)" />
    <node
        pkg="$(arg pkg)"
        type="$(arg type)"
        name="$(arg name)"
        args="$(arg args)"
        output="screen" />
</launch>
